### PR TITLE
UI fixes for checks section

### PR DESCRIPTION
**Pull request by Thomas1664**
- Make show/hide an actual button
- Add spacing between check name and `details`

### Before:

image

### After:

image

### CODE REVIEW

**Comment by Thomas1664** [comment]
Note that I reported the smaller ✔ on the wrapping check name as #4064

File: webviews/components/merge.tsx
Comment (daviddossett) [inline webviews/components/merge.tsx:69]: Let's avoid inline styles. What would be great here would be to extract the styles out of the header's new button group into something like `.small-button` so we can reuse them across both areas.

**Review by daviddossett — CHANGES_REQUESTED** [review]

**Review by alexr00 — APPROVED** [review]
Looks good to me, thank you for the PR! Will merge after @daviddossett has also signed off.

**Review by daviddossett — APPROVED** [review]
Looks good!

**Pull request merged**
